solr/solr-ref-guide/src/result-grouping.adoc

```diff
@@ -114,6 +114,11 @@ Setting this parameter to a number greater than 0 enables caching for result gro
 +
 Testing has shown that group caching only improves search time with Boolean, wildcard, and fuzzy queries. For simple queries like term or "match all" queries, group caching degrades performance.
 
+`group.skip.second.step`::
+This parameter can be set to `true` if only one document per group needs to be retrieved. Result Grouping executes two searches; if enabled this option will disable the second search improving the performance. By default the value is set to `false`. It can be set to `true` of if `group.limit` is 1, and `group.sort` fields list is a prefix of `sort` fields list (e.g., if `sort=id asc,name desc` and `group.sort=id asc` is fine, but  `sort=id asc,name desc` and `group.sort=name desc` is not). Also it cannot be used together with <<other-parsers.adoc#re-ranking-query-parser,Re-ranking Query Parser>>.
```

Comment:
Thanks for adding the detailed documentation here!

https://github.com/cpoerschke/lucene-solr/commit/5af6f12f8efb20f9866d1b4c1fc94747b689ccca has one `s/of if/if` edit and suggests to turn the `sort=id...` into `sort=price...` or similar in case the use of 'id' could lead to confusion since sorting by (document) id is perhaps a bit unusual, what do you think?

For the YouTube link, would it make sense to include wording with the description e.g. the `Learning to Rank: From Theory to Production` title so that readers have more clarity about what they are about to click on?

Reply:
Merged thanks!